### ISSUE TITLE
Target size and user-agent components that can't be styled

### ISSUE THREAD

**Issue by JamesCatt** [w3c/wcag#1904]
It might be good to include something in this SC explicitly stating that it does not apply to user-agent components that cannot be adequately styled by the author. For example:

- the options list of `<select>` dropdowns
- date picker for the HTML date input
- color picker for the HTML color input

This may seem obvious/implied for people who understand the HTML/CSS and the limitations around these, but I think it would be helpful to make it explicit for those who don't.

**Comment by patrickhlauke** [comment]
in essence, an exemption similar to the one in 2.5.5, but reworded from

> User Agent Control: The size of the target is determined by the user agent and is not modified by the author;

to something like

> User Agent Control: The size of the target is determined by the user agent and *cannot be modified* by the author;

**Comment by mraccess77** [comment]
Controlled by the user agent and not modified and "cannot be modified by the author" are two very different things.  Which one was the intent of the working group?

**Comment by jake-abma** [comment]
My memory says "not modified" and this already / also covers "cannot be modified by the author"

**Comment by jake-abma** [comment]
> "Controlled by the user agent and not modified "and "cannot be modified by the author"

All target sizes are in the basis / default controlled (determined) by the UA, and most can be modified.

BUT, all who are capable of being adjusted must be fixed, so the "Controlled by the user agent and not modified" must be modified if possible. This was the intent.

Leaving the only option open for the suggestion:

> User Agent Control: The size of the target is determined by the user agent and cannot be modified by the author;

**Comment by detlevhfischer** [comment]
@jake-abma 

> BUT, all who are capable of being adjusted must be fixed, so the "Controlled by the user agent and not modified" must be modified if possible.

I don't think the intention was to exempt only those elements that at a given point in time **cannot** be styled. Since this can change any time in new browser versions, int would be unclear whether content that could not be modified yesterday is suddenly found to fail today because now, it **can** be modified.
I think (but I am not positively sure) that the WG understanding was that if authors leave elements alone (i.e. apply no styling to select, checkboxes, radio inputs, etc.) then the target size requirement would be exempt.

**Comment by alastc** [comment]
Noting the last discussion on this is at: https://www.w3.org/2021/11/16-ag-minutes#item13

**Comment by alastc** [comment]
We've got this exception:
> User agent control: The size of the target is determined by the user agent and is not modified by the author;

And it is explained in the [understanding document](https://w3c.github.io/wcag/understanding/target-size-minimum.html).
